Change Makefile to include EasyBuild Eigen, Hypre and FlexiBLAS
and also some more environment variables coming from EasyBuild 
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/REEF3D-25.05_v2.orig/Makefile b/REEF3D-25.05_v2/Makefile
index 2176d15..9a43e88 100644
--- a/REEF3D-25.05_v2.orig/Makefile
+++ b/REEF3D-25.05_v2/Makefile
@@ -1,16 +1,16 @@
 BUILD    := ./build
 BIN    	 := ./bin
 TARGET   := REEF3D
-CXX      := mpicxx
+CXX      := ${MPICXX} 
 GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
 GIT_VERSION := "$(shell git describe --abbrev=8 --dirty --always --tags)"
 OBJ_DIR   := $(BUILD)
 APP_DIR   := $(BIN)
-HYPRE_DIR := /usr/local/hypre
-EIGEN_DIR := ThirdParty/eigen-3.3.8 
-CXXFLAGS := -w -std=c++11 -O3 -DVERSION=\"$(GIT_VERSION)\" -DBRANCH=\"$(GIT_BRANCH)\"
-LDFLAGS  := -L ${HYPRE_DIR}/lib/ -lHYPRE
-INCLUDE  := -I ${HYPRE_DIR}/include -I ${EIGEN_DIR} -DEIGEN_MPL2_ONLY 
+HYPRE_DIR := ${EBROOTHYPRE}
+EIGEN_DIR := ${EBROOTEIGEN}
+CXXFLAGS := -w -DVERSION=\"$(GIT_VERSION)\" -DBRANCH=\"$(GIT_BRANCH)\" ${CXXFLAGS} 
+LDFLAGS  := ${LDFLAGS} ${EBVARLIBSCALAPACK} -lHYPRE 
+INCLUDE  := -I${HYPRE_DIR}/include -I${EIGEN_DIR} -DEIGEN_MPL2_ONLY 
 SRC      := $(wildcard src/*.cpp)
 OBJECTS  := $(SRC:%.cpp=$(OBJ_DIR)/%.o)
 DEPENDENCIES := $(OBJECTS:.o=.d)
